R/espn_starters.R

```diff
@@ -87,19 +99,44 @@ ff_starters.espn_conn <- function(conn, weeks = 1:17, ...) {
   return(max_week)
 }
 
-.espn_week_starter <- function(week, conn) {
+.espn_week_matchup_periods_mapping <- function(settings) {
+
+  matchup_periods <- settings %>%
+    purrr::pluck("content", "settings", "scheduleSettings", "matchupPeriods")
+
+  ## first, flatten out the values.
```

Comment:
raw nested values look like this

```
$`16`[[1]]
[1] 17

$`16`[[2]]
[1] 18
```

it's easier to work with when it's like this

```
$`16`
[1] 17 18
```